src/pal/inc/pal.h

```diff
@@ -5453,6 +5453,19 @@ unsigned int _mm_getcsr(void);
 PALIMPORT
 void _mm_setcsr(unsigned int i);
 
+/******************* PAL functions for CPU capability detection *******/
+
+#ifdef  __cplusplus
+
+class CORJIT_FLAGS;
+
+PALIMPORT
+VOID
+PALAPI
+PAL_SetJitCpuCapabilityFlags(CORJIT_FLAGS &flags);
```

Comment:
The name of the function confuses me. The "Set" makes me feel the function is setting something inside PAL. I would prefer calling it PAL_GetJitCpuCapabilityFlags instead.
I would also prefer using pointer argument instead of reference, this is the convention of PAL functions. Or you can just return the CORJIT_FLAGS by value.